src/nft/utils/date.ts

```diff
@@ -0,0 +1,18 @@
+export const isValidDate = (date: number): boolean => {
+  const d = Date.parse(date.toString())
+  return isNaN(d) ? false : true
+}
+
+export const getTimeDifference = (eventTimestamp: string) => {
```

Comment:
This is not internationalized. Why not use a library for this?

Reply:
We have no yet started the work for internationalization for NFT components. For procedurally generated text such as getting time difference in either days, hours, or minutes is there a standard practice? When it is tackled which I believe is outside of the scope of this PR, should we return a Trans component in this case?

Reply:
Either return a trans component, or use a library which deals with internationalizing date/time strings. I believe we already use such a library in places like governance.